lib/createInlinePluginCreator.js

```diff
@@ -1,10 +1,6 @@
-const { writeFileSync } = require("fs");
 const debug = require("debug")("msr:inlinePlugin");
 const getCommitsFiltered = require("./getCommitsFiltered");
-const getManifest = require("./getManifest");
-const hasChangedDeep = require("./hasChangedDeep");
-const recognizeFormat = require("./recognizeFormat");
-const { get } = require("lodash");
+const { updateManifestDeps, resolveReleaseType } = require("./updateDeps");
```

Comment:
👍 had thought this code didn't really belong here when I was working on that refactor branch